src/main/scala/com/adform/sprint/mesos/SprintFramework.scala

```diff
@@ -190,8 +190,10 @@ class SprintFramework(containerRunManager: ContainerRunManager)(implicit context
 
     val containerInfo = ContainerInfo.newBuilder()
       .setDocker(dockerInfo)
-      .setType(ContainerInfo.Type.DOCKER)
-      .build()
+      .setType(containerRun.definition.container.`type` match {
+        case ContainerType.Docker => ContainerInfo.Type.DOCKER
+        case _ => ContainerInfo.Type.MESOS
```

Comment:
```suggestion
        case ContainerType.Mesos => ContainerInfo.Type.MESOS
```
maybe it's better to match explicitly so that possible future enum additions force you to update this